DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -22,10 +22,19 @@ export class SearchSortFilterAndPaginate {
 

```

Comment:
I can't comment it on the actual line, but have you check this works OK in IE?
```
// This is required to polyfill the new elements in IE
Details();
```
We aren't using the NHS details on this page, so it needs checking.

Reply:
Yeah seems to be working as expected.